Doc/library/datetime.rst

```diff
@@ -370,10 +375,10 @@ Notes:
    This is exact and cannot overflow.
 
 (3)
-   Division by 0 raises :exc:`ZeroDivisionError`.
+   Division by ``0`` raises :exc:`ZeroDivisionError`.
```

Comment:
This doesn't need code formatting.

Reply:
I'm not sure why some numbers don't need formatting

Reply:
I don't think any of them need formatting.

Reply:
The devguide states that [literals are marked up using double backticks](https://devguide.python.org/documentation/markup/#inline-markup):

> backquotes: \`\`text\`\` for code samples, variables, and literals.

Reply:
But for this particular example, I'd definitely spell it out:

```suggestion
   Division by zero raises :exc:`ZeroDivisionError`.
```